docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,14 @@
 title: Changelog
 ---
 
+## 13.6.6
+
+_Released 2/22/2024_
+
+**Bugfixes:**
+
+- Fixed an issue where `cypress verify` was failing for `nx` users. Fixes [#28982](https://github.com/cypress-io/cypress/issues/28982).
```

Comment:
```suggestion
- Fixed a regression introduced in [`13.6.5`](/guides/references/changelog#13-6-5) where `cypress verify` would fail for [`nx`](https://nx.dev/) users. Fixes [#28982](https://github.com/cypress-io/cypress/issues/28982).
```